src/vocutouts/workers.py

```diff
@@ -75,111 +85,86 @@ class TaskTransientError(Exception):
 
 
 @dramatiq.actor(queue_name="cutout", max_retries=1)
-def cutout_range(
+def cutout(
     job_id: str,
-    data_id: Dict[str, Union[str, int]],
-    ra_min: float,
-    ra_max: float,
-    dec_min: float,
-    dec_max: float,
+    data_ids: List[str],
+    stencils: List[Dict[str, Any]],
 ) -> List[Dict[str, Any]]:
-    """Implement a cutout for a specified range."""
+    """Perform a cutout.
+
+    This is a queue worker for the vo-cutouts service.  It takes a serialized
+    cutout request, converts it into a suitable in-memory format, and then
+    dispatches it to the scientific code that performs the cutout.  The
+    results are stored in a GCS bucket, and the details of the output are
+    returned as the result of the worker.
+
+    Parameters
+    ----------
+    job_id : `str`
+        The UWS job ID, used as the key for storing results.
+    data_ids : List[`str`]
+        The data objects on which to perform cutouts.  These are opaque
+        identifiers passed as-is to the backend.  The user will normally
+        discover them via some service such as ObsTAP.
+    stencils : List[Dict[`str`, Any]]
+        Serialized stencils for the cutouts to perform.  These are
+        JSON-serializable (a requirement for Dramatiq) representations of the
+        `~vocutouts.models.stencils.Stencil` objects corresponding to the
+        user's request.
+
+    Returns
+    -------
+    result : List[Dict[`str`, `str`]]
```

Comment:
If this is supposed to be consistent with the type annotation:
```suggestion
    result : List[Dict[`str`, `Any`]]
```